x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ssl/SSLConfigurationReloaderTests.java

```diff
@@ -153,15 +153,16 @@ public void testReloadingKeyStore() throws Exception {
      * Tests the reloading of SSLContext when a PEM key and certificate are used.
      */
     public void testPEMKeyConfigReloading() throws Exception {
+        assumeFalse("This fails with BCJSSE with 'certificate does not verify with supplied key'", inFipsJvm());
```

Comment:
I'll follow these up with appropriate issues once the troubleshooting is concluded

Reply:
Can we link them to an issue or something. I don't love having temporary `assumeFalse` in tests without a way to trace it back to an action.